contracts/src/interfaces/IWrappedNativeToken.sol

```diff
@@ -5,7 +5,8 @@
 
 pragma solidity 0.8.18;
 
-import {IERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/ERC20.sol";
+import {IERC20Upgradeable} from
```

Comment:
`IERC20Upgradeable` looks to be the exact same interface as `IERC20`, so I think we should should leave this as is, except maybe import it directly from the interface file (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/IERC20.sol)

I didn't realize were importing it from `ERC20.sol` previously

Reply:
`NativeTokenRemote` inherits from `IWrappedNativeToken` and `ERC20Upgradeable`, which is an `IERC20Upgradeable`. The `IERC20` and `IERC20Upgradeable` will conflict in their functions. One option is to add separate for `IWrappedNativeTokenUpgradeable` or similar.